### ISSUE TITLE
Feature request: webhooks/notifications

### ISSUE THREAD

**Issue by lukasmrtvy** [vmware-tanzu/velero#2428]
Would be supernice nice to implement custom notifications/webhooks if backup failed/succeded.
* Slack,MSTeams,Matrix,Mattermost 
* Custom webhook ( To support https://github.com/healthchecks/healthchecks for exampel .. )

Thanks

**Comment by sharkymcdongles** [comment]
I agree. I explored doing this via the plugins system, but it seems what is needed isn't  exposed there and any changes will need to be within Velero itself sadly.

**Comment by dsu-igeek** [comment]
Looks like a subset/dupe of general pre/post backup/restore hooks #3542, closing
